source/helpers/feature-link.ts

```diff
@@ -0,0 +1,3 @@
+export default function featureLink(id: string): string {
```

Comment:
I would put this in the index don't think it warrants its own file. JMO

@refined-github/maintainers?

Reply:
We have other one-liners like `browser-detection.ts`. I don't think it's a big deal.

What index are you referring to? AFAICT `options.tsx` only imports utils from `helpers` folder.